src/main/index.ts

```diff
@@ -30,7 +30,11 @@ if (app.commandLine.getSwitchValue("proxy-server") !== "") {
 }
 
 async function main() {
-  shellSync(app.getLocale());
+  try {
+    await shellSync(app.getLocale());
+  } catch (err) {
+    logger.warn("shellSync: ", err);
+  }
```

Comment:
Is it expected behaviour of `shell-env`? (throwing error)
Maybe it's better to put it inside `shell-sync.ts`?

Reply:
Yes, `shell-env` is expected to throw error if something happens. I'll move it to the other file.